accounts/support-rest-api.js

```diff
@@ -329,6 +335,48 @@ class SupportRestApi {
         }
     }
 
+    /**
+     * Get pairing info for miscellaneous services provided by the dojo
+     */
+    async getServices(req, res) {
```

Comment:
No need for async since there is no await
```suggestion
    getServices(req, res) {
```